File: js/1019.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
const tempo = Number(lines.shift());
let horas = 0;
let minutos = 0;
let segundos = 0;
let temporario = tempo

function contadorHoras() {
    horas++;
}

function contadorMinutos() {
    minutos++;
}

for (let i = 1; i <= tempo; i++) {
    if (temporario >= 3600) {
        contadorHoras()
        temporario -= 3600;
    }
    if (temporario >= 60) {
        contadorMinutos()
        temporario -= 60;
    }
}
segundos = temporario;

console.log(`${horas}:${minutos}:${segundos}`);

File: cs/1019.cs
int tempo = int.Parse(Console.ReadLine());
int horas = 0;
int minutos = 0;
int segundos = 0;
int temporario = tempo;

for (int i = 1; i <= tempo; i++)
{
    if (temporario >= 3600)
    {
        horas++;
        temporario -= 3600;
    }
    if (temporario >= 60) {
        minutos++;
        temporario -= 60;
    }

}
segundos = temporario;
Console.WriteLine($"{horas}:{minutos}:{segundos}");